src/main/java/org/broadinstitute/barclay/help/DocWorkUnitHandler.java

```diff
@@ -56,6 +56,13 @@ public String getDestinationFilename(final DocWorkUnit workUnit) {
     }
 
     /**
+     * Returns the destination prefix (no paths) that the handler would like
+     */
+    public String getJsonFilename(final DocWorkUnit workUnit) {
```

Comment:
Can you change the names of these new methods (this one and the DocWorkUnit one) so they use "JSON " instead of "Json" since its an acronym.